src/main/scala/temple/generate/kube/KubernetesGenerator.scala

```diff
@@ -28,20 +29,56 @@ object KubernetesGenerator {
       case GenType.StorageClaim => "PersistentVolumeClaim"
       case GenType.StorageMount => "PersistentVolume"
     }
-    val name = service.name + { if (isDb) "-db" else "" }
+    val name = service.name + (genType match {
+        case GenType.StorageClaim => "-db-claim"
+        case GenType.StorageMount => "-db-volume"
+        case _                    => if (isDb) "-db" else ""
```

Comment:
This line should likely be split into

```scala
case _ if isDb => "-db"
case _         => ""
```
```suggestion
        case if isDb              => "-db"
        case _                    => ""
```

It would also be nice if this could be stored in a variable (`suffix?`), `match` looks a bit odd in parens